### DISCUSSION TITLE
Short text field -> link (url) disappearing in endpoint output

### DISCUSSION THREAD

**Discussion by tseidler** [strapi/strapi#6904] [Legacy Topics]
**Describe the bug**
I have used the `Content-Types Builder` to build a content type to store 'affiliate links'. 
When I insert a very specific url the resulting value returned from the API / endpoint is "".

If I look in the Strapi admin panel everything seems fine. The URL is there.

**Steps to reproduce the behavior**

1. Go to `content type builder`
2. Create a new collection type
3. Name the collection type `test`
4. Add one field to the type, named `Link`
5. Add two of these test records
6. One with the link https://strapi.io
7. One with the link `https://www.awin1.com/cread.php?awinaffid=123`  (the number you enter doesn't matter, as long as it is a number)
8. Make sure public users can see localhost:1337/tests, set the correct permissions
9. Browse to localhost:1337:tests
10. Be amazed (see screenshots)

Now.. if you remove the number '123' or change anything else in the URL it'll work just fine.. as far as I have been able to test. 
I can't see any reason for this behaviour, so I'm hoping this is a Strapi bug and not just me ;-)

**Expected behavior**
I would expect the 'Link' to be there in the `/tests/` output, instead of an empty string.

**Screenshots**
![image](https://user-images.githubusercontent.com/5659459/86475555-32268600-bd45-11ea-96f3-c14800bacc7a.png)
![image](https://user-images.githubusercontent.com/5659459/86475621-51bdae80-bd45-11ea-9884-ddd34403820a.png)
![image](https://user-images.githubusercontent.com/5659459/86475674-6a2dc900-bd45-11ea-998b-de6ee25e0c3e.png)
![image](https://user-images.githubusercontent.com/5659459/86475727-803b8980-bd45-11ea-9f22-62198e98bcf3.png)

**System**

- Node.js version: 12.14.1
- NPM version: 6.13.7
- Strapi version: 3.0.5
- Database: sqlite
- Operating system: Windows 10, but using WSL

**Comment by tseidler** [comment]
So yeah.. I don't get it :). 
Any suggestions on how to debug this are welcome. 

And yeah.. '123' is not really my affiliate id :)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by tseidler** [reply]
Hi! While I appreciate your reply and moving it for me - how is this not a bug? :)

On certain input the output is not as you'd expect. The records in the database and admin panel are correct, but Strapi is stripping part of the output.

 IF this isn't a bug I would really like some insight as to why and where this is happening, so that I can fix it.

**Comment by tseidler** [comment]
It's been a few months, so I thought I'd bump this and see whether anyone has any input ;-)

**Reply by derrickmehaffy** [reply]
Hi @tseidler my apologies I should have replied much much sooner. I did test this on the latest version (v3.2.5) and couldn't reproduce it

![image](https://user-images.githubusercontent.com/8593673/97497697-2419f800-1928-11eb-8ee5-e787979dadd2.png)

**Reply by tseidler** [reply]
hmm, interesting! I'll give it a shot in the latest version **and** in a blank strapi setup :}
